spec/functional/resource/group_spec.rb

```diff
@@ -292,7 +292,15 @@ def create_group
     end
   end
 
-  let(:number) { rand(2000..9999) } # avoid low group numbers
+  let(:number) do
+    loop do
+      gid = rand(2000..9999) # avoid low group numbers
```

Comment:
Can we start at 8888?  That range should be good for testing and will avoid our AD ID range which often conflicts

Reply:
I originally was changing it to like `3000..6000` to avoid some groups on the rhel-7-power-le boxes where we failed recently because we hit an existing group. 

But what I did instead is to skip a gid if it exists already and pick a different one, so we don't have to play that game of trying to exclude any in use blocks. on the chance it hits one in use, it'll use another.